app/views/homescreen/blocks/_news.html.erb

```diff
@@ -1,15 +1,18 @@
 <%= render 'homescreen/blocks/header', title: t(:label_news_latest) %>
-
 <% unless @news.empty? %>
   <ul class="widget-box--arrow-links">
   <% @news.each do |news| %>
     <li class="-widget-box--arrow-multiline">
       <div>
-        <%= avatar(news.author, {class: 'avatar-mini'}) %>
+        <%= avatar(news.author, {class: 'avatar news-author-avatar'}) %>
         <%= link_to_project(news.project) + ': ' %>
         <%= link_to h(news.title), news_path(news) %>
```

Comment:
The angular widget has `div` wrapped around this block. So please add one here, too.